### ISSUE TITLE
Just Cause does not install on Python 3.6.9

### ISSUE THREAD

**Issue by Tanmay-Kulkarni101** [inovex/justcause#21]
I tried installing just cause on Python 3.6.9. I have already installed R and it seemed to have installed correctly with pip3. However, on execution, I got the following log:
```
python3
Python 3.6.9 (default, Nov  7 2019, 10:44:02)
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import justcause
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/tanmay/model_env/lib/python3.6/site-packages/justcause/__init__.py", line 4, in <module>
    from . import data
  File "/home/tanmay/model_env/lib/python3.6/site-packages/justcause/data/__init__.py", line 14, in <module>
    from . import sets
  File "/home/tanmay/model_env/lib/python3.6/site-packages/justcause/data/sets/__init__.py", line 24, in <module>
    from .ibm import load_ibm
  File "/home/tanmay/model_env/lib/python3.6/site-packages/justcause/data/sets/ibm.py", line 23, in <module>
    from ..frames import CausalFrame, Col
  File "/home/tanmay/model_env/lib/python3.6/site-packages/justcause/data/frames.py", line 4
    from __future__ import annotations
    ^
SyntaxError: future feature annotations is not defined
```

Can anyone help me out?

**Comment by FlorianWilhelm** [comment]
Hi @Tanmay-Kulkarni101, thanks for reporting. This is a bug on our side. It seems like this import was only introduced in Python 3.7. Could you try on Python 3.7?

@MaximilianFranz, could you make the import only if Python is greater or equal than 3.7?

**Comment by MaximilianFranz** [comment]
This is indeed because of the `annotations` feature only being available in `__future__` from Python 3.7 [(PEP563)](https://www.python.org/dev/peps/pep-0563/#enabling-the-future-behavior-in-python-3-7). 
I can add a conditional import, though `__future__` imports ought to remain at the beginning of the file, which is not the case, if I first import `sys` for checking the version 

```python 
import sys
if sys.version_info >= (3, 7):
    from __future__ import annotations
```

**Comment by MaximilianFranz** [comment]
Adding a `__future__` import anywhere but at the start of the file leads to a `SyntaxError`: 

```python
Python 3.7.6 | packaged by conda-forge | (default, Dec 26 2019, 23:46:52)
[Clang 9.0.0 (tags/RELEASE_900/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> if sys.version_info >= (3,7):
...     from __future__ import annotations
...
  File "<stdin>", line 2
SyntaxError: from __future__ imports must occur at the beginning of the file
```

**Comment by Tanmay-Kulkarni101** [comment]
I will try to run it with Python3,7 in that case. Thanks for your help.

**Comment by FlorianWilhelm** [comment]
This still isn't fixed and is also a bug in the version 0.3.1 release for Python 3.6 since `from __future__ import annotations` is still for instance in the `transport.py` module.

**Comment by FlorianWilhelm** [comment]
Thanks @MaximilianFranz, just tested it and importing `justcause` works under Python 2.6.10. I pushed to master.
